packages/react-components/src/components/OnboardingChecklist/types.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+export interface IChecklistItemProps {
+  id: string;
+  title: string;
+  description: string;
+  cta: React.ReactElement;
+  placeholder: React.ReactElement;
+}
+
+export interface ICheckListItem
+  extends Omit<IChecklistItemProps, 'placeholder'> {
+  isActive: boolean;
+  isChecked: boolean;
+  isLastElement: boolean;
+  onClick: (id: string) => void;
+}
+
+type CompleteItem = {
+  title: string;
+  titleLabel?: string;
+  placeholder?: React.ReactElement;
+  delay?: number;
+};
+
+export interface IOnboardingChecklistProps {
+  className?: string;
+  title: string;
+  titleLabel?: string;
```

Comment:
Better to rename it 
**GreetingText:** This name clearly indicates that the text is a greeting or welcome message.